tools/gcp-arch-viz/README.md

```diff
@@ -1,42 +0,0 @@
-# GCP Architecture Visualizer
```

Comment:
IMO it would be good to leave a README here that points to the tool's new home.

Reply:
Would removing the sub-directory but modifying the root README to point to the following work or do we always want to keep the sub-directory?

https://github.com/forseti-security/forseti-visualizer

Reply:
that makes sense just put link in root readme.

Reply:
Updated